server/src/db/index.ts

```diff
@@ -0,0 +1,48 @@
+import mongoose, {Schema} from 'mongoose';
+
+// Recruiter accounts
+let UserSchema : Schema = new Schema({
+  name: String,
```

Comment:
don't think we need a name

Reply:
Yeah if we aren't going to put it in scores/notes then we don't

Reply:
Oh yeah totally